nixos/doc/manual/installation/installing-usb.xml

```diff
@@ -9,13 +9,12 @@
   For systems without CD drive, the NixOS live CD can be booted from a USB
   stick. You can use the <command>dd</command> utility to write the image:
   <command>dd if=<replaceable>path-to-image</replaceable>
-  of=<replaceable>/dev/sdb</replaceable></command>. Be careful about specifying
+  of=<replaceable>/dev/sdX</replaceable></command>. Be careful about specifying
   the correct drive; you can use the <command>lsblk</command> command to get a
```

Comment:
Depending on how beginner-friendly we want to be we may want to be more specific: "Be careful about specifying the correct drive *instead of sdX*".

Reply:
At least sdX doesn't normally exist.